config/database.yml

```diff
@@ -8,6 +9,8 @@ development:
   username: safecast
   su_username: safecast
   password:
+  host: postgresql
+  port: 5432
```

Comment:
I haven't checked, but would this change break developers? I run Postgres on localhost.